gef.py

```diff
@@ -6321,7 +6321,7 @@ def do_invoke(self, argv):
 
 @register_command
 class GlibcHeapSetArenaCommand(GenericCommand):
-    """Display information on a heap chunk."""
+    """Display information on a heap set-arena."""
```

Comment:
heap set-arena doesn't really make sense. Also, the behaviour of this command is to set the address of `main_arena`, instead of displaying information on a heap chunk. It probably was an inaccuracy from an earlier commit.

Anyways, this modification is not in the scope of this PR so it would be better to revert this.